controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -392,6 +392,15 @@ void cli_bml::setFunctionsMapAndArray()
         " [<1(True) or 0(False)>] for force. Default value 1",
         static_cast<pFunction>(&cli_bml::set_wifi_credentials_caller), 2, 5, STRING_ARG, STRING_ARG,
         INT_ARG, INT_ARG, INT_ARG);
+    insertCommandToMap(
+        "bml_set_wifi_credentials_new", "<AL-MAC> <ssid> <network_key> <radio> <bss_type>",
```

Comment:
Don't bother with a new one. I understand that you want to avoid breaking stuff. However, at some point, we will anyway have to remove the old one, so we can just as well do it now.